site/src/request_handlers/dashboard.rs

```diff
@@ -137,13 +137,25 @@ pub async fn handle_dashboard(ctxt: Arc<SiteCtxt>) -> ServerResult<dashboard::Re
     let responses = ctxt
         .statistic_series(runtime_benchmark_query.clone(), aids.clone())
         .await?;
+
+    let mut ignore_runtime_benchmark = true;
```

Comment:
This bool is needed if there is some interpolated data *after* the first runtime benchmark result?

As above, please add a comment here that explains why do we need this handling (because historically, we don't have runtime benchmark data for old artifacts).